pkgs/development/python-modules/pythonix/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, ninja, meson, pkgconfig, gcc7, nixUnstable, isPy3k }:
+
+assert isPy3k;
+
+stdenv.mkDerivation rec {
```

Comment:
Please don't use `stdenv.mkDerivation` inside `python-packages.nix` without marking it as a Python module. It can be marked as a module with `toPythonModule`. This is needed e.g. for `python.buildEnv`.

Reply:
https://github.com/NixOS/nixpkgs/commit/b81de99c03784cc4e8f93932b948693e5bf204df